examples/get_authcode.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+import sys
+from bleak import BleakClient, BleakScanner
+from pyfreshintellivent import FreshIntelliVent
```

Comment:
Sorry, was a bit quick before I suggested the change...
```suggestion
import asyncio
import sys

from bleak import BleakClient, BleakScanner

from pyfreshintellivent import FreshIntelliVent

```